client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/Layout/Panels.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  SxProps,
+  Theme,
+  LocaleKey,
+  TypedTFunction,
+} from '@openmsupply-client/common';
+import { DraftRequestLine } from '../hooks';
+
+export type ValueInfo = {
+  label: string;
+  value?: number | null;
+  sx?: SxProps<Theme>;
+};
+
+export const getLeftPanel = (
```

Comment:
Can probably be re-used for requisition as well... will get to it when I get to it requisition refactor...